README.md

```diff
@@ -60,15 +64,69 @@ Your content structure must be mapped to the YooChoose domain model. This must b
 ### Tracking
 Events from the site needs to be sent to YooChoose so that recommendations can be adapted to visitors. Tracking can be setup in multiple ways, depending on anyone's constraints.
 
+`EzRecommendationBundle` delivers Twig extension which helps integrate tracking functionality into your site. All you need to do is place small snippet code somewhere in the HEAD section of your header template (if your bundle is built on top of the DemoBundle this is `page_head.html.twig`):
+
+```twig
+{{ yc_track_user(content.id) }}
+```
+
+Next step is to configure settings under the `recommender.included_content_types` parameter (see: `default_settings.yml` file delivered with this bundle).
+
+Here you can define for which content types tracking script will be shown.
+
 You can find more information on the YooChoose documentation, about [tracking in general](https://doc.yoochoose.net/display/PUBDOC/1.+Tracking+Events), and about the [generic asynchronous javascript tracker](https://doc.yoochoose.net/display/PUBDOC/Tracking+with+yc.js).
 
+###Displaying
+
+In order to allow displaying recommendations on your site you must add portions of scripts which will integrate recommender engine with your site.
+
+Implementation is very easy and can be performed in just a few steps (assuming that `EzRecommendationBundle` is properly configured and enabled in `EzPublishKernel.php`):
+
+* add additional JavaScript assets to your header template (if your bundle is built on top of the DemoBundle this is  `page_head_script.html.twig`):
+
+```twig
+{% javascripts
+    ...
+
+    '%kernel.root_dir%/../vendor/components/handlebars.js/handlebars.min.js'
+    '@EzSystemsRecommendationBundle/Resources/public/js/recommendationtemplaterenderer.js'
+    '@EzSystemsRecommendationBundle/Resources/public/js/recommendationrestclient.js'
+%}
+```
+
+* place dedicated Twig helper in place where you want to display recommendations:
+
+```twig
+{{ yc_show_recommendations(content.id, 'scenario_name', limit, 'template',
+    ['title', 'publishedDate', 'url', 'imgUrl', 'author']
+) }}
+```
+
+Parameter meanings:
+
+Parameter       | Type   | Description
+----------------|--------|------------
+`content.id`    | int    | this will be replaced by your current page ContentID
```

Comment:
`this is in content based views normally the twig variable holding the content id we want to get recommendations for`